assets/scss/components/_auto-complete.scss

```diff
@@ -29,6 +30,11 @@ Markup:
   </label>
 </fieldset>
 
+ComponentData:
```

Comment:
> This would make more sense as Example Data:

so `kss-node` doesn't strip spaces! so we end up with the variable `example data` (note the space) to use in the handlebars view. You ok with `ExampleData`? which will give us `exampledata` in the view